### ISSUE TITLE
Option to create new files in the currently open workspace / in a folder selected in options rather than in a default folder

### ISSUE THREAD

**Issue by ailintom** [Zettlr/Zettlr#5123]
### Description

As discussed under  #5116 as of 3.1.0-beta.2 Zettlr stopped creating new files (when clicking Ctrl+N or File -> New File ) in the currently opened Workspace and creates them in some generic folder instead (Documents folder on Windows). There is to my knowledge no option to select this default folder. This new approach completely breaks the usual routine for those who use Zettlr for Zettelkasten-styled note-taking. @nathanlesage  in https://github.com/Zettlr/Zettlr/issues/5116#issuecomment-2070544058 mentioned (also with reference to @sensologica  ) that it was done "to put it more in line with how users actually use the app", but I suppose that I am not the only one who uses Zettlr the old way — as a dedicated Zettelkasten knowledge base rather than a generalist Markdown editor. Hence, it would be nice to be able to select between the different behaviours in the preferences (creating new files in the active Workspace by default // creating new files in the Documents folder by default // creating new files in a folder selected by user in the preferences by default).  

Note: there is already a sort of a selector of a default folder in the Preferences under "Automatically create non-existing files in this folder when following internal links", but this option does not affect the "Ctrl+N"/"New file behaviour"

Thank you for your consideration

### Proposed Changes

- Add an option in preferences to select between creating new files in the active Workspace by default and creating new files in a default  or predefined folder

and / or

- Add an option in preferences to select the default folder for new files.

### Caveats

I see no caveats, but I would make the old behaviour the default option for old users

### Do you Wish to Attempt Implementing this Yourself?

No

### Zettlr Version

Beta (if applicable)

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Operating System Version

Windows 11

**Comment by hungvvu** [comment]
I'm experiencing the same workflow issue as well. I have Zettelkasten folders for different areas of my life (work, school courses, projects, etc.), and personally, I find this new behaviour of having Ctrl + N defaulting to create new files in the Documents folder makes the experience less seamless than before. To elaborate, instead of Ctrl + N --> Write, now I would have to either:

1. Ctrl + N --> Navigate to the correct folder --> Write
2. Right-click on the directory in the left-hand side file manager pane to create a new file --> Write

Basically, it adds another step (which could be quite lengthy depending on where your target directory is) to creating every new note. I thought this was a bug at first and almost opened a separate issue before I found this thread.

**Comment by lcsolano** [comment]
I support this too.

I've been using the CTRL+N shortcut as it's the perfect workflow.

**Comment by sensologica** [comment]
Thanks for your feedback, guys. I will discuss with @nathanlesage and we will address this. I agree with you, this behavior is not intuitive and requires more steps.

**Comment by alfs** [comment]
I had this problem in 3.2.0, I just upgraded to 3.2.1 and seems to be working as expected, at least on macOS. New files are created in current workspace, previously in Document folder.

Should this ticket be closed?

**Comment by nathanlesage** [comment]
Ah yes, correct, this can be closed now, since I restored the behavior I accidentally nuked for 3.1.0!
